src/main/java/net/jingles/enchantments/Commands.java

```diff
@@ -27,124 +27,148 @@
   public boolean onCommand(CommandSender sender, Command command, String name, String[] args) {
 
     if (args.length < 1) {
-      sender.sendMessage(ERROR + "Incorrect arguments.");
+      sender.sendMessage(ERROR + "Incorrect arguments: /enchantments help");
       return false;
     }
 
-    if (args[0].equalsIgnoreCase("enchant")) {
+    CustomEnchant enchant;
+    ItemStack held;
+    ItemMeta meta;
```

Comment:
I'm not sure how I feel about declaring variables when they might not be needed.